domain/src/main/java/org/reservationapplication/domain/repository/JDBCRepos/ReservationRepositoryJDBC.java

```diff
@@ -0,0 +1,197 @@
+package org.reservationapplication.domain.repository.JDBCRepos;
+
+import org.reservationapplication.domain.exeption.DatabaseErrorCode;
+import org.reservationapplication.domain.exeption.DatabaseException;
+import org.reservationapplication.domain.model.CoworkingSpace;
+import org.reservationapplication.domain.model.CoworkingSpaceType;
+import org.reservationapplication.domain.repository.ReservationRepository;
+import org.reservationapplication.logger.Loggers;
+import org.reservationapplication.domain.model.Reservation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.*;
+import java.util.*;
+
+@Repository
+public class ReservationRepositoryJDBC implements ReservationRepository {
```

Comment:
I suppose we agreed to leave only Spring Data Repo. In your case you will have cached repo also